raiden/api/rest.py

```diff
@@ -75,19 +75,26 @@
 
 URLS_V1 = [
     ('/address', AddressResource),
-    ('/channels', ChannelsResource),
-    ('/channels/<hexaddress:channel_address>', ChannelsResourceByChannelAddress),
-    ('/tokens', TokensResource),
-    ('/tokens/<hexaddress:token_address>/partners', PartnersResourceByTokenAddress),
-    ('/tokens/<hexaddress:token_address>', RegisterTokenResource),
-    ('/events/network', NetworkEventsResource),
+    ('/channels/<hexaddress:registry_address>', ChannelsResource),
```

Comment:
@LefterisJP should we introduce this changes now?

Reply:
no we should not.  Or if possible we could have this, but also using without a registry address should work. We should have an API where you can specify a registry but if you don't the default one would be used.

cc @carllin

Reply:
so, my suggestions lets revert this and just use the default registry for the `v1` of the rest api.

Reply:
As per our discussion, let's not have this here.

We should just keep the registry address as a client configuration option.